docs/translations.rst

```diff
@@ -3,12 +3,33 @@ Internationalization
 
 This page contains the most common techniques needed for managing CommCare HQ
 localization strings. For more comprehensive information, consult the
-`Django Docs translations page <https://docs.djangoproject.com/en/dev/topics/i18n/translation/>`_.
+`Django Docs translations page <https://docs.djangoproject.com/en/dev/topics/i18n/translation/>`_
+or `this helpful blog post <http://blog.bessas.me/post/65775299341/using-gettext-in-django>`_.
 
 
 Tagging strings in views
 ------------------------
 
+**TL;DR**: ``ugettext`` should be used in code that will be run per-request.
+``ugettext_lazy`` should be used in top-level code.
```

Comment:
that's what I mean.  Not sure how best to phrase it, and would welcome edits if you're so inclined.

Celery tasks I think don't have a locale unless specified.  For instance, scheduled reports do something like
```python
with localize(user.language):
    generate_report()
```